src/Wallabag/CoreBundle/Form/Type/NewCommentType.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Wallabag\CoreBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class NewCommentType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options)
+    {
+        $builder
+            ->add('content', 'textarea', array('required' => true,
+                'attr' => array('class' => 'materialize-textarea'),
```

Comment:
I have to call it if I want my textarea stylised in material theme, so I guess I'll have to implement it too in baggy.